examples/advanced/experiment-tracking/metrics-exchanger/jobs/hello-pt-wandb-metrics-exchanger/app/config/config_fed_server.json

```diff
@@ -0,0 +1,85 @@
+{
+  "format_version": 2,
+
+  "server": {
+    "heart_beat_timeout": 600
+  },
```

Comment:
this section is not needed, we can remove it.

We can extract common components/sections from these jobs later and make it a new job template.